source/_docs/wordpress-redis.md

```diff
@@ -31,6 +31,13 @@ This method will store object cache values persistently in Redis while preservin
  ```
  terminus wp <site>.<env> -- plugin install wp-redis
  ```
+ 
+ For site networks, you will need to specify the site URL by adding that to the command:
+ 
+  ```
```

Comment:
I meant the codeblock backticks, but it turns out it was GitHub's diff view messing with me. I'm going to delete this comment to hide my shame, but your email will forever remember it.